libs/NfcCoreLib/inc/phLibNfc.h

```diff
@@ -723,6 +741,41 @@ NFCSTATUS phLibNfc_SE_SetMode ( phLibNfc_Handle              hSE_Handle,
 
 /**
 * \ingroup grp_lib_nfc
+* \brief Sets power and link control configuration for the eSE secure element type.
+*
+* This function configures SE to different power mode.
+*
+* -# If mode is set to #phLibNfc_SE_NfccDecides then the NFCC will be responsible for powering
+* the eSE.
+* -# If mode is set to #phLibNfc_SE_PowerSupplyAlwaysOn then the NFCC will keep the eSE power
+* supply always on.
+* -# If mode is set to #phLibNfc_SE_NfccComLinkActiveOnPowerOn then the NFCC will keep the eSE
+* link always on.
+* -# If mode is set to #phLibNfc_SE_PowerNfccLinkAlwaysOn then the NFCC will keep the eSE
+* power supply and the link always on.
+*
+* \param[in]  hSE_Handle           Secure element handle
+* \param[in]  ePowerAndLinkModes   eSE power and link modes
+* \param[in]  pSE_PowerAndLinkControl_Rsp_cb   Pointer to response callback
+* \param[in]  pContext             Client context which will be included in
+*                                  callback when the request is completed.
+*
+* \retval   #NFCSTATUS_PENDING             SE power and link control transaction started
+* \retval   #NFCSTATUS_SHUTDOWN            Shutdown in progress
+* \retval   #NFCSTATUS_NOT_INITIALISED     LibNfc is not yet initialized
+* \retval   #NFCSTATUS_INVALID_HANDLE      Invalid handle
+* \retval   #NFCSTATUS_INVALID_PARAMETER   Invalid parameter
+* \retval   #NFCSTATUS_REJECTED            Invalid request
+* \retval   #NFCSTATUS_FAILED              Request failed
+*/
+NFCSTATUS phLibNfc_SE_PowerAndLinkControl(phLibNfc_Handle              hSE_Handle,
+    phLibNfc_PowerLinkModes_t  ePowerAndLinkModes,
```

Comment:
Is the function needs to be exposed to upper layer?

Reply:
It has been a ask from Microsoft to expose this function in LibNfc.
The idea behind is to provide an IOCTL to be used by SEMgrSvc only rather than upon SCARD_xxx event.